native/exe/SageLauncher/launcher.cpp

```diff
@@ -875,8 +895,8 @@ int WINAPI WinMain( HINSTANCE hInst, 	/*Win32 entry-point routine */
 				break;
 			}
 		} while (shouldRestartJVM(FALSE, bClient));
-		delete [] appPath;
-		delete [] newCmdLine;
+		delete [] &appPath;

```

Comment:
Remove both of these lines, you're trying to deallocate stack memory like it was heap memory. Stack memory doesn't need to be explicitly deallocated.